src/Rector/Class_/CommandPropertyToAttributeRector.php

```diff
@@ -118,34 +206,104 @@ public function refactor(Node $node): ?Node
             }
         }
 
-        $node->attrGroups[] = $this->createAttributeGroup($defaultName, $defaultDescription);
-
-        return $node;
+        return $defaultDescription;
     }
 
-    private function createAttributeGroup(string $defaultName, ?string $defaultDescription): AttributeGroup
+    private function resolveAliases(Class_ $class): ?Array_
     {
-        $attributeGroup = $this->phpAttributeGroupFactory->createFromClass(self::ATTRIBUTE);
+        $commandAliases = null;
+        $classMethod = $class->getMethod('configure');
+        if (! $classMethod instanceof ClassMethod) {
+            return null;
+        }
+        $this->traverseNodesWithCallable($class->stmts, function (Node $node) use (&$commandAliases) {
+            if (! $node instanceof MethodCall) {
+                return null;
+            }
 
-        $attributeGroup->attrs[0]->args[] = new Arg(new String_($defaultName));
+            if ($node->isFirstClassCallable()) {
+                return null;
+            }
 
-        if ($defaultDescription !== null) {
-            $attributeGroup->attrs[0]->args[] = new Arg(new String_($defaultDescription));
-        }
+            if (! $this->isObjectType($node->var, new ObjectType('Symfony\Component\Console\Command\Command'))) {
+                return null;
+            }
 
-        return $attributeGroup;
+            if (! $this->isName($node->name, 'setAliases')) {
+                return null;
+            }
+
+            /** @var Arg $arg */
+            $arg = $node->args[0];
+            if (! $arg->value instanceof Array_) {
+                return null;
+            }
+
+            $commandAliases = $arg->value;
+
+            $parentNode = $node->getAttribute(AttributeKey::PARENT_NODE);
+            if ($parentNode instanceof MethodCall) {
+                $parentNode->var = $node->var;
+            } else {
+                $this->removeNode($node);
+            }
+
+            return NodeTraverser::STOP_TRAVERSAL;
+        });
+
+        return $commandAliases;
     }
 
-    private function getValueFromProperty(Property $property): ?string
+    private function resolveHidden(Class_ $class): ?ConstFetch
     {
-        if (count($property->props) !== 1) {
+        $commandHidden = null;
+        $classMethod = $class->getMethod('configure');
+        if (! $classMethod instanceof ClassMethod) {
             return null;
         }
-        $propertyProperty = $property->props[0];
-        if ($propertyProperty->default instanceof String_) {
-            return $propertyProperty->default->value;
+        $this->traverseNodesWithCallable($class->stmts, function (Node $node) use (&$commandHidden) {
```

Comment:
Oh, it need to be `$classMethod->stmts`:

```suggestion
        $this->traverseNodesWithCallable((array) $classMethod->stmts, function (Node $node) use (&$commandHidden) {
```

Reply:
++